google/cloud/pubsub/samples/samples.cc

```diff
@@ -124,8 +124,6 @@ void PleaseIgnoreThisSimplifiesTestingTheSamples() {
 
 void CreateTopic(google::cloud::pubsub::TopicAdminClient client,
                  std::vector<std::string> const& argv) {
-  //! [START pubsub_quickstart_create_topic]
-  //! [START pubsub_create_topic] [create-topic]
```

Comment:
We have to leave the DevRel snippets in place until we update our internal docs. They are referenced by:

https://cloud.google.com/pubsub/docs/create-topic#pubsub_create_topic-cpp

Reply:
The DevRel regions are the `[START ...]` ones.

Reply:
Done